src/core_modules/capture-core/components/TemplateSelector/TemplateSelector.component.tsx

```diff
@@ -87,9 +85,14 @@ const TemplateSelectorPlain = (props: Props) => {
         );
     });
 
+    const savedListsHeader = i18n.t('Saved lists in this program') as ReactNode;
+    const quickAccessText = i18n.t('Saved lists offer quick access to your most used views in a program.') as ReactNode;
+    const noSavedListsText = i18n.t('There are no saved lists in this program yet, create one using the button below.') as ReactNode;
+    const createSavedListText = i18n.t('Create saved list') as ReactNode;
```

Comment:
Let's keep these in the same place as they were. However, I see that you have a typescript error. Can you update the `declarations.d.ts` file and say that this `.t()` function returns a ReactNode? It should not give you any errors then.

Reply:
Move these back to their original placement. The definitions.d.ts file should state that the `i18n.t()` function should return any now. That's the correct behavior